updated copy

alaizaz opened the pull request:
Current text
Streamline and scale with the Aqovia A1 Autonomic Platform. Moving Telcos forward with Artificial Intelligence based operations.

Amended
Streamline and scale with the Aqovia A1 Autonomic Platform. Moving companies forward with Artificial Intelligence based operations.

Current text
WHY USE AN AUTONOMIC PLATFORM ?
Aqovia A1 - Autonomic Platform is applicable to various solutions and sectors, including Telco, Enterprise, Finance, Trade and Government.

Amended
WHY USE AN AUTONOMIC PLATFORM ?
Aqovia A1 - Autonomic Platform is applicable to various solutions and sectors, including Green Energy, Telco, Enterprise, Finance, Trade and Government.

alaizaz committed:
updated copy

Current text
Streamline and scale with the Aqovia A1 Autonomic Platform. Moving Telcos forward with Artificial Intelligence based operations.

Amended
Streamline and scale with the Aqovia A1 Autonomic Platform. Moving companies forward with Artificial Intelligence based operations.

Current text
WHY USE AN AUTONOMIC PLATFORM ?
Aqovia A1 - Autonomic Platform is applicable to various solutions and sectors, including Telco, Enterprise, Finance, Trade and Government.

Amended
WHY USE AN AUTONOMIC PLATFORM ?
Aqovia A1 - Autonomic Platform is applicable to various solutions and sectors, including Green Energy, Telco, Enterprise, Finance, Trade and Government.

kinaly reviewed (changes requested):
It feels like you have amended the "artefact" file as opposed to the source file, therefore the changes may be lost in future builds.
There is a short explanation on the readme file of the repo on how to build (but may be too succinct when you are not familiar with the process).

alaizaz committed:
Added a class with 50% division of columns

Replied the previous class which divdes column by 59-41. New class does division by 50-50 to be used on renewable-energy.html page

alaizaz committed:
Adding a new page to the website

alaizaz committed:
Images of Wind-Turbine

This image is used on "What we do" page. 
svg wasn't rendering properly so I have used PNG. 
If Kinaly can take up this responsibility of replacing PNG with SVG, then SVG is included.

alaizaz committed:
Added, new type of solution

alaizaz committed:
Video on renewable energy page

alaizaz committed:
Add files via upload

alaizaz committed:
Add files via upload

alaizaz committed:
Add files via upload

alaizaz committed:
Add files via upload

alaizaz commented:
Can you please approve it, files have been moved to the relevant folder. This has to published for an event on Thursday.

alaizaz committed:
Update renewable-energy.html

kinaly reviewed (changes requested):
There are still files that have been amended in the wrong places and would be deleted in future builds.
Let's take some time together to review and show you how to amend the site :)
